app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'lisa.haitz@uc.edu'
+  default from: 'uclappdev@uc.edu'
```

Comment:
I think you set up a default from address in your ENV variables.  You can definitely use that here.  Something like:
class ApplicationMailer < ActionMailer::Base
default_sender = ENV['MY_DEFAULT_SENDER'] || 'uclappdev@uc.edu'
default from: default_sender

Adding in the || symbol will make your code more robust in case something goes wrong with the ENV file or someone forgets to add the variable.  Since this isn't technically a secret, it's not "wrong" to hardcode like you did, but putting in an ENV call here will allow for easier changes in the future.